### PR TITLE
Build containerized version for Google Cloud Run

### PR DESCRIPTION
**Pull request by peterschen**
This PR adds a `Dockerfile` together with a `cloudbuild.yaml` and associated changes to build a containerized version of the solution that can be deployed to Google Cloud Run.

### CODE REVIEW

**Comment by jpassing** [comment]
Apache 2 license headers are missing in all files

**Comment by peterschen** [comment]
I've added the licensing headers.

**Review by jpassing — APPROVED** [review]
Lgtm. I'd still prefer to merge the two Python files, but that can happen in a separate PR.

File: ad-joining/register-computer/docker.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import main
```
[the remark this answers is not in the document]
Comment (peterschen) [reply ad-joining/register-computer/docker.py]: This is a shim to continue to support GCF. As we now have decided to drop support for GCF I'll merge.

**Pull request merged**